scripts/tests/chiptest/__init__.py

```diff
@@ -85,9 +85,10 @@ def _GetManualTests() -> Set[str]:
 
 def _GetFlakyTests() -> Set[str]:
     """List of flaky tests, ideally this list should become empty."""
-    return {
-        "Test_TC_OO_2_4.yaml"
-    }
+    return_value = set()
```

Comment:
Adding blocking comment so that PR will be merged only when I am around for a few days to monitor CI

Additionally I might update this PR to run on CI for this PR multiple times without merging

Reply:
Passed twice already in this CI so I am just going to enable for everyone hoping to see if perhaps it is actually resolved since there were other changes to on/off recently, or hopefully get some logs from flakes that happen in other CI runs